lockdown/core.py

```diff
@@ -7,9 +7,14 @@
 
 DEF_GUILD = {
     "channels": [],
+    "roles": [],
     "lockdown_message": None,
     "unlockdown_message": None,
-    "confirmation_message": False,
+    "locked": False,
+    "vc_channels": [],
+    "embed_set": False,
```

Comment:
- role dict (nondefault settings and list for futures)
- lock_check (locked var) to prevent double runs